packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -296,6 +296,63 @@ class UiKitView extends StatefulWidget {
   State<UiKitView> createState() => _UiKitViewState();
 }
 
+/// Embeds arbitrary HTML in the Widget hierarchy.
```

Comment:
This sounds like I can add HTML to any app running on any platform (e.g. Android, iOS, etc.). Is that true? If not, can we clearly document where this is available?